src/orders/build.gradle

```diff
@@ -45,6 +45,26 @@ dependencies {
   }
 
   testImplementation 'com.openpojo:openpojo:0.8.12'
+ 
+  // Newly added dependencies for OCI SDK
+  implementation 'org.springframework.boot:spring-boot-starter-actuator'
```

Comment:
Are all these deps really required ?

Reply:
+1 (Why?)

Reply:
they are taken from shipping service as is.. bit of trial and error here. I'll revisit this.